filing_history_descriptions.yml

```diff
@@ -1075,9 +1075,10 @@ description:
     'limited-liability-partnership-members-register-information-on-withdrawal-from-the-public-register' : "**Members' register** information at {date} on withdrawal from the public register"
     'limited-liability-partnership-members-usual-residential-address-register-information-on-withdrawal-from-the-public-register' : "**Members' usual residential address register** information at {date} on withdrawal from the public register"
     'limited-liability-partnership-persons-with-significant-control-register-information-on-withdrawal-from-the-public-register' : "**Persons' with significant control register** information at {date} on withdrawal from the public register"
-    'confirmation-statement' : "**Confirmation statement** made on {made_up_date}"
     'confirmation-statement-with-no-updates' : "**Confirmation statement** made on {made_up_date} with no updates"
     'confirmation-statement-with-updates' : "**Confirmation statement** made on {made_up_date} with updates"
+    'confirmation-statement-with-made-up-date' : "**Confirmation statement** made on {made_up_date}"
```

Comment:
Upon merging this line into develop, there won't be any entries that match this within chs-backend. Do you have accompanying CHS-backend changes to go with them too? I can see a cs01-mappings and a cs01-catch-all-mappings branch in chs-backend. If they are to go in as well am happy to review them too. If they do not go in, that will make this line redundant.

Reply:
I have found your ch-backend PR in a different slack channel, and referenced here: https://github.com/companieshouse/chs-backend/pull/747 
ignore comment above as changes there address this.